dagger.nim

```diff
@@ -0,0 +1,22 @@
+import pkg/chronos
+
+type
+  DaggerPeer* = ref object
+
+proc newDaggerPeer*: DaggerPeer =
```

Comment:
I would keep all the terminology as IPFS in this repo.

It would be helpful to mention that Dagger is an umbrella name for a set of components, one of them is a storage engine, which is based on IPFS, other components such as the incentivization(?) or long term storage system(?), etc, will have their own names.

Reply:
I'll address this in a later commit.

Reply:
Should be resolved with the latest couple of commits